src/Indice.Features.Cases.App/src/app/features/dashboard/dashboard.component.ts

```diff
@@ -31,6 +33,12 @@ export class DashboardComponent implements OnInit {
             this.isAdmin = this.authService.isAdmin();
         });
         this.metaItems = [];
+        this.canvases = settings.canvases.split(',');
     }
 
+    showCanvas(id: string) {
+        return settings.canvases === ''
+            ? true
+            : this.canvases.find(canvas => canvas === id);
```

Comment:
εδώ νιώθω πως κάνουμε την ανάγνωση πιο δύσκολη και δυσνόητη. Δηλαδή βασίζεσαι στο γεγονός πως η find() επιστρέφει item (truthy) αν βρει αλλιώς undefined (falsy) αλλά νομίζω αυτό προκαλεί λίγο μπέρδεμα στο διάβασμα:

νομίζω η some() είναι καλύτερο fit:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some
επίσης και το ternary μου φαίνεται υπερβολή:

`return settings.canvases === '' || this.canvases.some(canvas => canvas === id);`